include/translated-object.php

```diff
@@ -422,6 +412,50 @@ protected function should_update_translation_group( $id, $translations ) {
 		return ! empty( array_diff_assoc( $translations, $old_translations ) );
 	}
 
+	/**
+	 * Returns an array of valid translations for multiple objects.
+	 *
+	 * @since 3.8
+	 *
+	 * @param int[] $ids Array of object IDs.
+	 * @return int[] An associative array of translations with language code as key and translation ID as value.
+	 *
+	 * @phpstan-return array<non-empty-string, positive-int>
+	 */
+	protected function get_objects_translations( array $ids ) {
+		$translations = $this->get_raw_objects_translations( $ids );
+
+		return $this->validate_objects_translations( $translations );
+	}
+
+	/**
+	 * Returns an unvalidated array of translations for multiple objects.
+	 * It is generally preferable to use `get_objects_translations()`.
+	 *
+	 * @since 3.8
+	 *
+	 * @param int[] $ids Array of object IDs.
+	 * @return array[] An associative array of translations arrays with language code as key and translation ID as value.
```

Comment:
The method returns `int[][]` and the description is not exact. This is not an associative array but an array of associative arrays, and it describes only the second level key. See the phpstan-return.

Reply:
Done in 052478dd4cc9fda0e0c909ff163f09151e52ea85